docs/001_develop/03_client-capabilities/023_custom-components/03_state-management.mdx

```diff
@@ -81,21 +107,25 @@ export class MyButton extends GenesisElement {
 <my-button label="Cancel"></my-button>
 ```
 
-- Here, the label attribute binds directly to the button's text via `${x => x.label}`.
-- If label is set to "Cancel" in HTML `(<my-button label="Cancel">)`, the button will display "Cancel".
-- When the label value is changed dynamically, the button text will automatically update.
+##### Key Points:
+
+> - Here, the label attribute binds directly to the button's text via `${(x) => x.label}`.
+> - If label is set to "Cancel" in HTML `(<my-button label="Cancel">)`, the button will display "Cancel".
+> - When the `label` value is changed dynamically, the button text will automatically update.
 
 ### Interpolated Attribute Values
 
 In cases where an attribute value is partially static and partially dynamic, we can combine static and dynamic parts using interpolated expressions. This approach is useful for creating customized button labels or CSS classes.
 
-Suppose we want to add a counter to the button label that shows the number of times it has been clicked. We could use an expression to combine the static text ("Clicked: "), and a dynamic value, such as the clickCount property:
+For instance, consider `MyButton` component. You can use interpolation to dynamically construct an `aria-label` that combines a static prefix ("Perform action:") with a dynamic property (`action`). Similarly, the button’s visible label can be updated based on the label property:
 
 ```typescript
 @customElement({
   name: "my-button",
-  template: html` <button
-    aria-label="${(x) => `Perform action: ${x.action}`}"
+  template: html<MyButton>`<button
+    aria-label="${(x) =>
+      `Perform action: ${x.action}`}" //textual description for assistive technologies
```

Comment:
I think move the comment outside of the template as I think it might cause syntax errors if the user copy and pastes the example